src/System.Runtime/tests/System/TypeTests.netcoreapp.cs

```diff
@@ -36,5 +36,42 @@ public void IsSZArray(Type type, bool expected)
         {
             Assert.Equal(expected, type.IsSZArray);
         }
+
+        private static IEnumerable<object[]> VariableBoundArrayOrNotTypes()
+        {
+            // Not arrays
+            yield return new object[] { typeof(void), false };
+            yield return new object[] { typeof(int), false };
+            yield return new object[] { typeof(int[]).MakeByRefType(), false };
+            yield return new object[] { typeof(TypeTests), false };
+            yield return new object[] { Type.GetType("System.Int32"), false };
+            yield return new object[] { typeof(Outside<int>.Inside<string>), false };
+
+            // Arrays, but SZ arrays
+            yield return new object[] { Type.GetType("System.Int32[]"), false };
+            yield return new object[] { typeof(int[]), false };
```

Comment:
For the MakeArrayType(1) case, please enclose it in a guard like this:

```
    if (!PlatformDetection.IsNetNative) // "Multidim rank 1 arrays not supported: https://github.com/dotnet/corert/issues/3331"
    {
        yield return ... MakeArrayType(1)
    }

```

This will save us having to do it as we bring up the corefx tests on .NET Native :-)